branch/starter/main.go

```diff
@@ -2,61 +2,42 @@ package main
 
 import (
 	"context"
-	"flag"
+	"fmt"
 	"time"
 
-	"github.com/pborman/uuid"
-	"go.temporal.io/temporal/client"
-	"go.uber.org/zap"
-
 	"github.com/temporalio/temporal-go-samples/branch"
+	"go.temporal.io/temporal/client"
 )
 
 func main() {
-	logger, err := zap.NewDevelopment()
```

Comment:
I understand the intent but I don't know if it is good or not. Now worker and starter use different approach but from other side starter output is better now.